README.md

```diff
@@ -18,13 +18,54 @@ Once the plugin is installed on the Kong instance, it can be configured via the
 {
     "name": "health-apis-token-validator",
     "config": {
-        "verification_url": "{verification_url}",
-        "verification_timeout": {verification_timeout},
-        "api_key": "{api_key}"
+        "verification_url": "{verification-url}",
+        "verification_timeout": {verification-timeout},
+        "api_key": "{api-key}",
+        "static_token": "{static-token}",
+        "static_icn": "{static-icn}"
     },
     "enabled": true
 }
 ```
 
-- [Local development](development/README.md)
 
+
+## health-apis-static-token-handler Kong Plugin
+
+A Kong plugin to return a static access token used for customer testing
+
+> Luarocks must be installed to build.
+
+Run `luarocks pack health-apis-static-token-handler-0.1-1.rockspec` to package the plugin into a Lua "rock" which can then be copied and installed on a Kong instance.
+
+
+### Configuration
+
+Once the plugin is installed on the Kong instance, it can be configured via the Admin port.  Replace config entries with the correct values for your environment.
+
+```
+{
+    "name": "health-apis-token-validator",
+    "config": {
+        "static_refresh_token": "{static-refresh-token}",

```

Comment:
@patrickvinograd @edpaget - Just as our plugin that validates an incoming access token allows a special static value through, we now have this plugin to allow a static *refresh token* value through (which simply returns the static access token). 

This flow is used by Pluto in their testing harness. 

As such, we will need two more values added to credstash. I will DM @patrickvinograd on Slack providing values for:

- {static-refresh-token}
- {static-scopes}